utils/s2n_safety.h

```diff
@@ -117,7 +120,7 @@ extern pid_t s2n_actual_getpid();
 extern int s2n_constant_time_equals(const uint8_t * a, const uint8_t * b, uint32_t len);
 
 /* Copy src to dst, or don't copy it, in constant time */
-extern int s2n_constant_time_copy_or_dont(const uint8_t * dst, const uint8_t * src, uint32_t len, uint8_t dont);
+extern int s2n_constant_time_copy_or_dont(uint8_t * dst, const uint8_t * src, uint32_t len, uint8_t dont);
```

Comment:
confused myself but this was done to match the actual definition in s2n_safety.c